### ISSUE TITLE
[rngh/2.20.0 + rn0.76-rc.6] Unresolved reference: getEventDispatcher

### ISSUE THREAD

**Issue by jbrodriguez** [software-mansion/react-native-gesture-handler#3169]
### Description

i can't compile my app on android, works on ios

this is on rn 0.76.0-rc.6

### Steps to reproduce

install react-native-gesture-handler on a project
try to run on android
throws an error

```
<snipped>node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt:11:19 Unresolved reference: getEventDispatcher
```

### Snack or a link to a repository

https://snack.expo.dev/G3drm9e9VrExnn-LRLvMA

### Gesture Handler version

2.20.0

### React Native version

0.76.0-rc.6

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native (without Expo)

### Architecture

Fabric (New Architecture)

### Build type

Debug mode

### Device

Android emulator

### Acknowledgements

Yes

**Comment by haelsisy** [comment]
Same with me

**Comment by m-bert** [comment]
It was already fixed in #3166
